.cbmc-batch/jobs/aws_array_list_get_at/aws_array_list_get_at_harness.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <aws/common/array_list.h>
+#include <proof_helpers/make_common_data_structures.h>
+
+#define MAX_ITEM_SIZE 2
+#define MAX_INITIAL_ITEM_ALLOCATION (UINT64_MAX / MAX_ITEM_SIZE) + 1
+
+/**
+ * Coverage: 1.00 (118 lines out of 118 statically-reachable lines in 17 functions reached)
+ * Runtime: 0m5.428s
+ *
+ * Assumptions:
+ *     - a valid non-deterministic aws_array_list bounded by initial_item_allocation and item_size
+ *     - non-deterministic initial_item_allocation <= MAX_INITIAL_ITEM_ALLOCATION
+ *     - non-deterministic item_size <= MAX_ITEM_SIZE
+ *     - non-deterministic length <= initial_item_allocation
+ *     - aws_array_list_get_at == AWS_OP_SUCCESS
+ *                                 -> list->data != NULL
+ *                                 -> list->length > index
+ *     - all parameters remain the same after this operation
+ */
+void aws_array_list_get_at_harness() {
+    struct aws_array_list *list;
+    ASSUME_BOUNDED_ARRAY_LIST(list, MAX_INITIAL_ITEM_ALLOCATION, MAX_ITEM_SIZE);
+
+    void *val = malloc(list->item_size);
+    size_t index = nondet_size_t();
```

Comment:
Why invoke a function to assign a nondet value when CBMC itself will assign a nondet value to an uninitialized local?

Reply:
For the sake of clarity for those who are not familiar with CBMC. Should I remove it?